calculator.py

```diff
@@ -1,7 +1,23 @@
 # Objective: create a calculator
+a = int(input('enter first  number:'))
```

Comment:
can we not name this as a valid variable names instead of just a and b like inputVariable1 and inputVariable2?